<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="topLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QFrame" name="verticalFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Отрезки</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="pointsTableView">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>230</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="cutterLabel">
               <property name="text">
                <string>Отсекатель</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cutterValueLabel">
               <property name="text">
                <string>Не задан</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="addPointFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTabWidget" name="tabWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="LineTab">
                <attribute name="title">
                 <string>Отрезок</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="addLineLabel">
                   <property name="text">
                    <string>Координаты концов отрезка</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="addPointInputGrid_2" columnminimumwidth="50,0,0">
                   <item row="1" column="1">
                    <widget class="QSpinBox" name="lineBeginX_SB">
                     <property name="maximum">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QSpinBox" name="lineBeginY_SB">
                     <property name="maximum">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="lineBeginLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>(Xн, Yн)</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="addPointInputGrid" columnminimumwidth="50,0,0">
                   <item row="1" column="1">
                    <widget class="QSpinBox" name="lineEndX_SB">
                     <property name="maximum">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QSpinBox" name="lineEndY_SB">
                     <property name="maximum">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="lineEndLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>(Xк, Yк)</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="addLineButton">
                   <property name="text">
                    <string>Добавить отрезок</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="cutterRectTab">
                <attribute name="title">
                 <string>Отсекатель</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="cutterRectLabee">
                   <property name="text">
                    <string>лев. верхний и пр. нижний углы</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="addPointInputGrid_4" columnminimumwidth="65,0,0">
                   <item row="1" column="1">
                    <widget class="QSpinBox" name="rectTLPointX_SB">
                     <property name="maximum">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QSpinBox" name="rectTLPointY_SB">
                     <property name="maximum">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="rectTLPointLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>(Xлв, Yлв)</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QGridLayout" name="addPointInputGrid_3" columnminimumwidth="65,0,0">
                   <item row="1" column="1">
                    <widget class="QSpinBox" name="rectRBPointX_SB">
                     <property name="maximum">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QSpinBox" name="rectRBPointY_SB">
                     <property name="maximum">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="rectRBPointLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>(Xпн, Yпн)</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="addCutterRectButton">
                   <property name="text">
                    <string>Добавить отсекатель</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="colorChoiceFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="colorFrameLabel">
               <property name="text">
                <string>Цвет закраски</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="colorChoiceGrid">
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Цвет отрезков</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Цвет отсекателя</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="cutterColorChoiceBtn">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Цвет результата</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="linesColorChoiceBtn">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QPushButton" name="resColorChoiceBtn">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="bottomLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="cutoffButton">
            <property name="text">
             <string>Отсечь</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cleanScreenButton">
            <property name="text">
             <string>Очистить экран</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="graphicsViewFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGraphicsView" name="graphicsView"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>справка</string>
    </property>
    <addaction name="action"/>
    <addaction name="aboutTaskAction"/>
   </widget>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="fileImportAction">
   <property name="text">
    <string>импорт модели</string>
   </property>
  </action>
  <action name="aboutTaskAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="fileExportAction">
   <property name="text">
    <string>экспорт модели</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Управление</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
